src-tauri/tauri.conf.json

```diff
@@ -17,6 +17,26 @@
                 "execute": true,
                 "sidecar": true,
                 "scope": [
+                    {
+                        "name": "arpaname",
+                        "cmd": "arpaname",
+                        "args": true
+                    },
+                    {
+                        "name": "amass",
+                        "cmd": "amass",
+                        "args": true
+                    },
+                    {
```

Comment:
Although it is good that you restored which as a scoped command, this is not part of this task and is being solved by pull request #864 
```suggestion
                   
```